torchaudio/transforms.py

```diff
@@ -444,3 +452,136 @@ def __call__(self, x_mu):
 
     def __repr__(self):
         return self.__class__.__name__ + '()'
+
+
+class Pad(object):
```

Comment:
This class seems very similar to PadTrim (https://github.com/pytorch/audio/blob/7ee1d389bb07a77e3fcbc4afcfa59fbb96306d04/torchaudio/transforms.py#L75).
Is there any difference?

Reply:
does not look like. They bot end up using the same `torch.functional.pad` function.